README.md

```diff
@@ -1,27 +1,42 @@
+# Temporal Go SDK
+[![Build Status](https://badge.buildkite.com/ce6df3b1a8b375270261ae70fb2d2756af298fef3a0dac4d20.svg?theme=github&branch=master)](https://buildkite.com/temporal/temporal-go-client) [![Coverage Status](https://coveralls.io/repos/github/temporalio/temporal-go-sdk/badge.svg?branch=master)](https://coveralls.io/github/temporalio/temporal-go-sdk?branch=master) [![PkgGoDev](https://pkg.go.dev/badge/go.temporal.io/sdk)](https://pkg.go.dev/go.temporal.io/sdk)
 
-[Temporal](https://github.com/temporalio/temporal) is a distributed, scalable, durable, and highly available orchestration engine used to execute asynchronous long-running business logic in a scalable and resilient way.
+The Temporal Go SDK provides a framework for Temporal Application development in the Go language.
+The SDK contains the following tools:
 
-"Temporal Go SDK" is the framework for authoring workflows and activities using Go language.
+- A Temporal Client to communicate with a Temporal Cluster
+- APIs to use within your Workflows
+- APIs to create and manage Worker Entities and Worker Processes
 
-## How to use
+**How to use**
 
-Clone this repo into the preferred location.
+Add the [Temporal Go SDK](https://github.com/temporalio/sdk-go) to your project:
 
 ```bash
-git clone git@github.com:temporalio/sdk-go.git
+go get -u go.temporal.io/sdk@latest
 ```
 
-or
+Or clone the Go SDK repo to your preferred location:
 
 ```bash
-go get -u go.temporal.io/sdk
+git clone git@github.com:temporalio/sdk-go.git
 ```
+**Are there executable code samples?**
+
+You can find a complete list of executable code samples in the [samples library](/docs/samples-library/#go), which includes Temporal Go SDK code samples from the [temporalio/samples-go](https://github.com/temporalio/samples-go) repo.
+Additionally, each of the Go SDK Tutorials in the [Temporal docs site](https://docs.temporal.io/docs/go) is backed by a fully executable template application.
+
+**Where is the Go SDK technical reference?**
 
-See [samples](https://github.com/temporalio/samples-go) to get started.
+The [Temporal Go SDK API reference](https://pkg.go.dev/go.temporal.io/sdk) is published on [pkg.go.dev](https://pkg.go.dev/go.temporal.io/sdk)
 
-Documentation is available [here](https://docs.temporal.io). 
-You can also find the API documentation [here](https://pkg.go.dev/go.temporal.io/sdk).
+
+**Links**
+
+- [Contribution Guidelines](CONTRIBUTING.md)
+- [Temporal Server repository](https://github.com/temporalio/temporal)
+- [Temporal Documentation site](https://docs.temporal.io)
+- [MIT License](LICENSE)
 
 ## Using a custom logger
```

Comment:
Wonder if you want to move this somewhere else. It's strange to just have this one sample in the README and not the dozens of other custom things you could create.

Reply:
Will remove it in this PR, and add it to the docs in this PR: https://github.com/temporalio/documentation/pull/866